tests/ert_tests/data/test_integration_data.py

```diff
@@ -289,11 +305,24 @@ def create_summary_observation():
     return observations
 
 
-def create_general_observation():
+def create_general_observation(force_failure=False):
     observations = ""
-    index_list = list(range(1, 2001))
-    random.shuffle(index_list)
-    index_list = [index_list[i : i + 4] for i in range(0, len(index_list), 4)]
+    index_list = np.array(range(1, 2001))
+
+    sane = False
+    while not sane:
+        random.shuffle(index_list)
+        index_list = [index_list[i : i + 4] for i in range(0, len(index_list), 4)]
+
+        # Must avoid idx 2000 in the first block because the test
+        # using these observations skips that block
+        sane = 2000 not in index_list[0]
+
+    # The caller wants us to generate a set of observations which fails...
+    if force_failure:
+        idx = [idx for idx in range(len(index_list)) if 2000 in index_list[idx]][0]
+        index_list[idx], index_list[0] = index_list[0], index_list[idx]
```

Comment:
```suggestion
        idx = np.where(index_list == 2000)
        index_list[idx], index_list[0] = index_list[0], index_list[idx]
```

Reply:
Not sure  the change in line 323 does what it must do...  At this point index_list is a list of lists (inner lists contain 4 elements each) - we need the (first and only) index of the outer list where the inner list contains 2000.

Reply:
Ah. Never mind then😅